installation/routines/install.sh

```diff
@@ -8,6 +8,7 @@ install() {
   init_git_repo_from_tardir
   setup_jukebox_core
   setup_mpd
+  setup_spotifyd
```

Comment:
Does this mean spotifyd is always enabled, even when the user doesn’t select Spotify?